src/load/azext_load/tests/latest/constants.py

```diff
@@ -126,6 +135,7 @@ class LoadTestConstants(LoadConstants):
     REGIONAL_LOAD_CONFIG_TEST_ID = "regional-load-config-test-case"
     LOAD_TEST_KVREF_ID = "loadtest-kvrefid-case"
     LOAD_TEST_SPLITCSV_ID = "loadtest-splitcsv-case"
+    LOAD_TEST_ADVANCED_URL = "loadtest-advanced-url-case"
```

Comment:
LOAD_TEST_ADVANCED_URL -> is it used as test id ? may be add a suffix?

Reply:
updated